src/Features/CSharp/Portable/Completion/CompletionProviders/ObjectCreationCompletionProvider.cs

```diff
@@ -56,16 +55,16 @@ protected override SyntaxNode GetObjectCreationNewExpression(SyntaxTree tree, in
             return CSharpSyntaxContext.CreateContext(document.Project.Solution.Workspace, semanticModel, position, cancellationToken);
         }
 
-        protected override async Task<ImmutableArray<ISymbol>> GetPreselectedSymbolsWorker(SyntaxContext context, int position, OptionSet options, CancellationToken cancellationToken)
+        protected override async Task<ImmutableArray<(ISymbol symbol, CompletionItemRules rules)>> GetPreselectedSymbolsWorker(SyntaxContext context, int position, OptionSet options, CancellationToken cancellationToken)
         {
             var result = await base.GetPreselectedSymbolsWorker(context, position, options, cancellationToken).ConfigureAwait(false);
             if (result.Any())
             {
-                var type = (ITypeSymbol)result.Single();
+                var type = (ITypeSymbol)result.Single().symbol;
                 var alias = await type.FindApplicableAlias(position, context.SemanticModel, cancellationToken).ConfigureAwait(false);
                 if (alias != null)
                 {
-                    return ImmutableArray.Create(alias);
+                    return ImmutableArray.Create((alias, CompletionItemRules.Default));
```

Comment:
Yes, that's fine - I was just taking a pass through PRs, and I didn't know if this one was ready to merge or not, so I took a look through it.